src/domain/auth/adGroups.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Checks if the user belongs to any of the allowed Active Directory (AD) groups.
+ *
+ * @param {string[]} ad_groups - An array of AD group names the user belongs to.
+ * @returns {boolean} - Returns `true` if AD filtering is enabled and the user belongs to at least one of the allowed AD groups, otherwise `false`.
```

Comment:
The comment is misleading. This doesn't check if the AD filtering is enabled.

Reply:
Thanks for noticing. I first put the AD filtering check in this function but removed it and forgot the comment.